moderator.lua

```diff
@@ -178,7 +178,7 @@ function HandleAntiOchCommand(Split, Player)
 			cRoot:Get():BroadcastChat("...lobbest thou thy Holy Hand Grenade of Antioch towards thy foe,")
 			cRoot:Get():BroadcastChat("who being naughty in My sight, shall snuff it.")
 		end
-		Player:GetWorld():SpawnPrimedTNT(LookPos.x, LookPos.y, LookPos.z, 35)
+		Player:GetWorld():SpawnPrimedTNT(Vector3d(LookPos.x, LookPos.y, LookPos.z), 35)
```

Comment:
Wouldn't `Vector3d(LookPos)` work too?

Reply:
Not with the current approach. LookPos is a table.